src/index.js

```diff
@@ -2,5 +2,9 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import DocumentViewer from "./components/documentViewer";
+import templateRegistry from "./templates"; // why this dependency
```

Comment:
If we make `DocumentViewer` a component, we won't need the ./templates folder already. Right now, those in the template folder are sample templates. The object is a key-value map from template name to the actual templates themselves.

Reply:
yeah exactly, I just dont want to implement this here for the moment as I planned to create a real implementation in open-attestation :) (so I took the shortcut just to show an example on how to implement it)

idea is later to update this renderer to use whatever we came use as a solution in open-attestation =)